sources/Launcher/DYNRobustnessAnalysisLauncher.cpp

```diff
@@ -40,16 +44,24 @@
 #include <JOBDynModelsEntryFactory.h>
 #include <JOBModelerEntry.h>
 #include <DYNMacrosMessage.h>
+#include <DYNTrace.h>
 
+#include <config.h>
+#include <gitversion.h>
+
+#include "../config_algorithms.h"
```

Comment:
I don't really like this relative path in include but could be addressed in another issue

Reply:
It was done already like that before this issue !

But I can generate this file (config_algorithms.h) as well as the other one (gitversion_algorithms.h) directly in the Launcher folder ?

Reply:
Yes I know it was like that before but it would be better if we address this here or in another issue. If generating in the Launcher folder is the best solution then why not

Reply:
I propose to do this in another PR as it is independent